packages/input/src/index.vue

```diff
@@ -222,6 +222,9 @@ export default defineComponent({
       type: Object,
       default: () => ({}),
     },
+    maxlength: {
```

Comment:
@wen-haoming but you made the type `string` acceptable here, so I thought you need to do a string parse before doing actual arithmetic operations

Reply:
Since we do not want to break the old functionality as passing that to `<input />` which accepts both `string` and `number` type. I do think that you need to perform a `Number` or `parseInt` for `maxlength` though.

Reply:
I think so 😄